sqlite3/CMakeLists.txt

```diff
@@ -17,7 +17,15 @@ add_definitions(
   -DSQLITE_ENABLE_DBSTAT_VTAB
 )
 
-if(NOT WIN32)
+if(WIN32)
+  foreach(flag_var
+          CMAKE_C_FLAGS CMAKE_C_FLAGS_DEBUG CMAKE_C_FLAGS_RELEASE
+          CMAKE_C_FLAGS_MINSIZEREL CMAKE_C_FLAGS_RELWITHDEBINFO)
+    if(${flag_var} MATCHES "/MD")
+      string(REGEX REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}")
```

Comment:
Ah, yes! It didn't work because I forgot to swap the `/MD` to `/MT` for CMake's C flags.